kubernetes/wes-gps-server.yaml

```diff
@@ -36,7 +36,6 @@ spec:
             privileged: true
           resources:
             limits:
-              cpu: 50m
               memory: 10Mi
```

Comment:
when looking at w023 it was seen that gps has 1000m as a limit.. maybe from the default cpu limit (that we want want to get rid of)

Reply:
Yeah some WES pods that already has its limit set, `update-stack.sh` does not restart them. We will need to do some cleanups after applying the PR. I will put a note on this and address later.